src/backend/tools/retrieval/logs_retriever.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, List, Dict
+from elasticsearch import Elasticsearch
+from backend.tools.retrieval.base import BaseRetrieval
+import os
+from backend.services.logger import get_logger
+
+cohere_api_key = os.environ.get("COHERE_API_KEY")
+logger = get_logger()
+
+
+class LogsRetriever(BaseRetrieval):
+    def __init__(
+            self,
+            host_url: str,
+            index: str,
+            deployment_name: str,
+    ) -> None:
+        self.index = index
```

Comment:
Suggestion: Rather than instantiating `LogsRetriever()` with the index and host, can you add these as params on the class, eg :

```
class LogsRetriever(BaseRetrieval):
    index = os.environ.get(.., None)
    host = os.environ.get(.., None)
```

Then your `__init__()` method would use `self.index` and `self.host`,

And your classmethod 
`is_available()` should check `return all[self.index is not None, self.host is not None]`